src/amo/components/SearchContextCard/index.js

```diff
@@ -154,7 +204,16 @@ export class SearchContextCardBase extends React.Component<InternalProps> {
           }
           break;
         default:
-          if (query) {
```

Comment:
It seems like the `category` variations are missing for this `default` case. I can see that was the case before, but is there a reason for it? I'm wondering if maybe we should fix that now.

Reply:
categories are per add-on type, so there can't be a category for the default (unknown/missing type) case.